cmds/wget/wget.go

```diff
@@ -37,6 +43,13 @@ func wget(arg string, w io.Writer) error {
 	if resp.StatusCode != 200 {
 		return fmt.Errorf("non-200 HTTP status: %d", resp.StatusCode)
 	}
+
+	w, err := os.Create(fileName)
```

Comment:
Just one question, maybe the arg should still be an os.Writer, not a string? Sorry, was in a hurry yesterday. It might make tests easier. Up to you.

Reply:
*io.Writer, and I'd prefer that too

Reply:
io.Writer as argument assumes that the file is already created when wget() is called.
Deferring the creation after the http.Get() allows to create the output file only when the download succeeds and not on errors.

Reply:
sounds good to me.